lib/node_modules/@stdlib/_tools/changelog/generate/README.md

```diff
@@ -44,13 +44,23 @@ The function returns an object with the following properties:
 -   **content**: Markdown formatted changelog.
 -   **releaseType**: release type (`null` if changelog is for a non-release).
 
-To generate a changelog for an upcoming release, provide a valid release type as the second argument.
+The function accepts the following `options`:
+
+-   **releaseType**: a release type for which to generate the changelog.
+-   **format**: whether to generate a `grouped` changelog where commits are grouped by package or `aggregated` one.
```

Comment:
```suggestion
-   **format**: changelog format. Must be one of the following:

    -   `'grouped'`: group commits by package.
    -   `'aggregated'`: TODO:add description.
```

Given the sub-list, you'll need to insert blank lines to satisfy Markdown linting.